components/wrapper/Wrapper.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function Wrapper({ styles, children }) {
```

Comment:
is it okay to name the wrapper just `Wrapper`? like this one applies the padding and the max width to the page. what if someone creates another wrapper that does other things? shouldn't the wrappers be named according to what they do?